### DISCUSSION TITLE
Add 'all' content type for message_handler

### DISCUSSION THREAD

**Discussion by vovkapultik** [eternnoir/pyTelegramBotAPI#1803] [Ideas]
If you want to handle all types of content it's quite worthless job to do, I'd like you to consider adding 'all' in a content_types options as aiogram's (async botapi wrapper) devs have done

**Comment by Badiboy** [comment]
utils.content_type_media
utils.content_type_service

PS. Not an issue.

**Reply by coder2020official** [reply]
+Agree
